R/install-github.r

```diff
@@ -98,6 +106,10 @@ remote_download.github_remote <- function(x, quiet = FALSE) {
   } else {
     auth <- NULL
   }
+  ## if repo uses submodules warn user
```

Comment:
Thanks! Sorry for missing the indent and if-paren style with this PR. 





I've made the changes, and also fixed indent style in changes the tests






- Jaime

On Tue, Apr 21, 2015 at 6:51 AM, Hadley Wickham <notifications@github.com>
wrote:

>> @@ -98,6 +106,10 @@ remote_download.github_remote <- function(x, quiet = FALSE) {
>>    } else {
>>      auth <- NULL
>>    }
>> +  ## if repo uses submodules warn user
> 1. I think the comment is extraneous (given how simple the code is)
> 2. Can you please use a space after `if`
> 3. `warning()` looks like it's indented too much
> 4. I always use `call. = FALSE` with warning
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hadley/devtools/pull/751/files#r28778939